blueflood-core/src/main/java/com/rackspacecloud/blueflood/types/DataType.java

```diff
@@ -32,20 +32,21 @@ public DataType(String type) {
     public final static DataType DOUBLE = new DataType("D");
     public final static DataType BOOLEAN = new DataType("B");
     public final static DataType BIGINT = new DataType("BI");
+    public final static DataType NUMERIC = new DataType("N");
 
     public static DataType getMetricType(Object metricValue) {
         if (metricValue instanceof String) {
             return STRING;
         } else if (metricValue instanceof Integer) {
-            return INT;
+            return NUMERIC;
```

Comment:
Can we consolidate these blocks into a single `metricValue instanceOf Number`? Ref: https://github.com/rackerlabs/blueflood/pull/421#discussion_r23633118